packages/react/src/components/Icon/FontIcon.test.tsx

```diff
@@ -10,6 +10,24 @@ describe('FontIcon', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  it('renders named FontIcon correctly', () => {
+    const component = renderer.create(<FontIcon iconName="CompassNW" aria-label="compass" />);
```

Comment:
I know I mentioned snapshot tests before, but thinking about it more I wonder if instead or in addition, we ought to have explicit tests for how the text is applied and what other attributes are applied. Let me know if it's unclear what I mean by that (or how to do it--not sure what your level of background is with enzyme/jest tests) and I can go into more detail.

Reply:
I updated the tests to use explicit assertions, and kept a couple of the snapshots w/ names in there as well. Let me know what you think!